src/visual/visual.ts

```diff
@@ -6,30 +6,24 @@ const logger = require('@blackbaud/skyux-logger');
 const PixDiff = require('pix-diff');
 const protractor = require('protractor');
 
-let nextUniqueId = 0;
-
 export abstract class SkyVisual {
+
+  /**
+   * Captures a new screenshot and compares it with an existing screenshot.
+   * @param selector The CSS selector of the element to capture.
+   * @param config The configuration to use for the screenshot capture.
+   */
   public static compareScreenshot(
-    selector = 'body',
```

Comment:
Why are we removing the default `body` selector?

Reply:
Because the config is no longer optional. We can't have an optional parameter before an optional one.